package.json

```diff
@@ -13,9 +13,12 @@
   },
   "devDependencies": {
     "browserify": "latest",
+    "chai": "^3.5.0",
     "derequire": "latest",
-    "watchify": "latest",
-    "uglifyjs": "latest"
+    "jsdom": "^9.12.0",
+    "mocha": "^3.2.0",
+    "uglifyjs": "latest",
```

Comment:
I'd recommend [`babili`](https://github.com/babel/babili) and others (`uglifyjs` is quite outdated and unmaintained)